cloud/aws/kops-asg.md

```diff
@@ -57,6 +57,10 @@ kubect apply -f px-spec.yaml
 
 {% include k8s-monitor-install.md %}
 
+### Corelating EBS volumes with Portworx nodes
+
+{% include asg/cli.md %}
```

Comment:
For k8s, let's use kubectl to run pxctl. So you can share the output from the includes but not the pxctl command.

Reply:
makes sense. Will do the change.

Reply:
done